dashboard/modules/reporter/reporter_agent.py

```diff
@@ -300,10 +316,40 @@ def _get_all_stats(self):
         }
 
     @staticmethod
-    def _record_stats(stats):
+    def _record_stats(stats, cluster_stats):
         records_reported = []
-
         ip = stats["ip"]
+        redis_address = stats["redis_address"]
+
+        # -- Instance count of cluster --
+        if "autoscaler_report" in cluster_stats and str(ip) == str(
+                redis_address):
```

Comment:
see the `self.is_head_node` comment above

Reply:
done